docs/api-reference/leaksdb/endpoints/get-credentials-by-domain.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "List by Domain"
+api: "GET https://api.flare.io/leaksdb/v2/credentials/by_domain/{domain}"
+---
+
+Returns a list of accounts matching the domain provided.
+
+Results are returned in ascending order by id.
+
+<ResponseExample>
+
+```json Response Example
+{
+    "items": [
+        {
+            "domain": "scatterholt.com",
+            "hash": "B@dPassw0rd",
+            "id": 33880703907,
+            "identity_name": "ryan.howard@scatterholt.com",
+            "imported_at": "2024-07-22T19:25:52.893439+00:00",
+            "known_password_id": null,
+            "source": {
+                "breached_at": null,
+                "description_en": "Collection of multiple combo lists (emails and passwords) exchanged on illicit networks.",
+                "description_fr": "Collection de multiples listes \"combos\" (adresses courriel et mots de passe) \u00e9chang\u00e9es sur des r\u00e9seaux illicites.",
+                "id": "combolists",
+                "is_alert_enabled": true,
+                "leaked_at": null,
+                "name": "Combolists"
+            },
+            "source_id": "combolists"
+        },
+        {
+            "domain": "scatterholt.com",
+            "hash": "1qaz2wsx",
+            "id": 33880703906,
+            "identity_name": "ryan.howard@scatterholt.com",
+            "imported_at": "2024-07-22T19:25:52.893439+00:00",
+            "known_password_id": null,
+            "source": {
+                "breached_at": null,
+                "description_en": "Collection of multiple combo lists (emails and passwords) exchanged on illicit networks.",
+                "description_fr": "Collection de multiples listes \"combos\" (adresses courriel et mots de passe) \u00e9chang\u00e9es sur des r\u00e9seaux illicites.",
+                "id": "combolists",
+                "is_alert_enabled": true,
+                "leaked_at": null,
+                "name": "Combolists"
+            },
+            "source_id": "combolists"
+        }
+    ],
+    "next": "WyJjb20uc2NhdHRlcmhvbHQiLCAxNjczNjg4ODg4NV0"
+}
+```
+
+</ResponseExample>
+
+## Paging
+
+This endpoint supports the 
+[Flare standard paging pattern <Icon icon="book" size={16} />](/concepts/paging).
+
+## Path Parameters
+
+<ParamField path="domain" type="string" required>
+  The domain you want to lookup.
```

Comment:
`lookup` is a noun, whereas `look up` would be the verb form

Reply:
fixed

Reply:
I also removed the "you"